drltrace/scripts/gen_drlttrace_config.py

```diff
@@ -0,0 +1,379 @@
+"GEOID":"GEOID", "FINDEX_SEARCH_OPS":"FINDEX_SEARCH_OPS", "FINDEX_INFO_LEVELS":"FINDEX_INFO_LEVELS",
+"COMPUTER_NAME_FORMAT":"COMPUTER_NAME_FORMAT", "CHAR_INFO":"CHAR_INFO", "ULONGLONG":"ULONGLONG",
+"NLS_FUNCTION":"NLS_FUNCTION", "DWORD_PTR":"DWORD_PTR", "STREAM_INFO_LEVELS":"STREAM_INFO_LEVELS",
+"SECURITY_INFORMATION":"SECURITY_INFORMATION", "NORM_FORM":"NORM_FORM",
+"JOBOBJECTINFOCLASS":"JOBOBJECTINFOCLASS", "INPUT_RECORD":"INPUT_RECORD",
+"HEAP_INFORMATION_CLASS":"HEAP_INFORMATION_CLASS", "GEOTYPE":"GEOTYPE", "GEOCLASS":"GEOCLASS",
+"FILE_SEGMENT_ELEMENT":"FILE_SEGMENT_ELEMENT", "FILE_INFO_BY_HANDLE_CLASS":"FILE_INFO_BY_HANDLE_CLASS",
+"DWORDLONG":"DWORDLONG","EXCEPTION_POINTERS":"EXCEPTION_POINTERS", "_EXCEPTION_POINTERS":"_EXCEPTION_POINTERS",
+"WOW64_CONTEXT":"WOW64_CONTEXT", "UILANGUAGE_ENUMPROCW":"UILANGUAGE_ENUMPROCW",
+"UILANGUAGE_ENUMPROCA":"UILANGUAGE_ENUMPROCA", "TIMEFMT_ENUMPROCW":"TIMEFMT_ENUMPROCW",
+"TIMEFMT_ENUMPROCEX":"TIMEFMT_ENUMPROCEX", "TIMEFMT_ENUMPROCA":"TIMEFMT_ENUMPROCA",
+"NUMBERFMTW":"NUMBERFMTW",  "LOGICAL_PROCESSOR_RELATIONSHIP":"LOGICAL_PROCESSOR_RELATIONSHIP",
+"LOCALE_ENUMPROCW":"LOCALE_ENUMPROCW", "LOCALE_ENUMPROCEX":"LOCALE_ENUMPROCEX",
+"LOCALE_ENUMPROCA":"LOCALE_ENUMPROCA", "LATENCY_TIME":"LATENCY_TIME", "LANGUAGEGROUP_ENUMPROCW":"LANGUAGEGROUP_ENUMPROCW",
+"LANGUAGEGROUP_ENUMPROCA":"LANGUAGEGROUP_ENUMPROCA", "GROUP_AFFINITY":"GROUP_AFFINITY", "GEO_ENUMPROC":"GEO_ENUMPROC",
+"EXECUTION_STATE":"EXECUTION_STATE", "DYNAMIC_TIME_ZONE_INFORMATION":"DYNAMIC_TIME_ZONE_INFORMATION",
+"CURRENCYFMTW":"CURRENCYFMTW", "CONTEXT":"CONTEXT", "CONSOLE_CURSOR_INFO":"CONSOLE_CURSOR_INFO", "CHAR":"char",
+"APPLICATION_RECOVERY_CALLBACK":"APPLICATION_RECOVERY_CALLBACK", "HRESULT": "HRESULT", "PCNZCH": "char*",
+"LPPROCESS_INFORMATION": "PROCESS_INFORMATION*", "LPINT": "int*", "LSTATUS":"LSTATUS",
+"DEP_SYSTEM_POLICY_TYPE":"DEP_SYSTEM_POLICY_TYPE", "CURRENCYFMTA":"CURRENCYFMTA", "va_list": "va_list"}
+
+
+tokens_ignore_prefix = {"WINAPI", "WINBASEAPI", "APIENTRY", "virtual", "DECLSPEC_NORETURN",
+                        "WINNORMALIZEAPI"}
+
+PATH = "results/" # path where the output of headers_parser.py was saved
+
+unknown_types = dict() # we use this dictionary to save and print a list of unknown types
+
+def parse_internal_parentesses(str):
+    while True:
+        end = str.find(")")
+        if end == -1:
+            break
+        start = str.rfind("(", 0, end)
+        if start == -1:
+            break
+        new_str = str[:start]
+        new_str += str[end+1:]
+        str = new_str
+    return str
+
+def get_type(type_str):
+    return str(types_map.get(type_str, None))
+
+def parse_prefix(line):
+    '''
+    The function searches for a WinAPI return type in the string specified
+    @in line - a string with a type to find
+    @out - returning type
+    '''
+    if "//" in line or "Boolean" in line or "&&" in line:
+        return None
+    if ")" in line:
```

Comment:
I expect nested parens because remove_internal_parentheses will handle it; the remove_internal_parentheses has a loop.